main/src/main/java/cgeo/geocaching/utils/ImageUtils.java

```diff
@@ -212,7 +212,12 @@ private static BitmapDrawable scaleBitmapTo(@NonNull final Bitmap image, final i
         final ImmutableTriple<Integer, Integer, Boolean> scaledSize = calculateScaledImageSizes(image.getWidth(), image.getHeight(), maxWidth, maxHeight, minWidth, minHeight);
 
         if (scaledSize.right) {
-            result = BitmapCompat.createScaledBitmap(image, scaledSize.left, scaledSize.middle, null, true);
+            if (image != null && image.getConfig() == null) {
```

Comment:
The `image` parameter is anotated with `@NonNull`, so we need no null check here.

Reply:
That's a sensitive topic. Unfortunately these annotations (`@NonNull` and `@Nullable`) are only informative and not enforced by the JVM or Android. That's why they are frequently violated both by our own code as well as library or even Android system code. I am thus advocating leaving this null-check in.

Reply:
Understand.
But then you should do the check before line 212. There you have already access to the image object: `image.getWidth()`.
And on the other side, with `image==0`, you would use the else branch below (line 212).

Reply:
That is quite true. Thanks for the observation, I will change this

Reply:
@eddiemuc 
FYI: I haven't merged this PR yet since you mentioned you wanted to move the `image == null` check further up

Reply:
I now removed the null check completely, as initially suggested by @bekuno. He was right in pointing out that if image is null the code will fail in earlier lines. Since this is not happening in production, the null check is superfluous. Thanks for the review! And sorry for late reaction, I am kinda bussy currently unfortunately